parse/lex.go

```diff
@@ -397,7 +397,7 @@ func lexName(l *lexer) stateFn {
 		r rune   // current rune
 	)
 
-	for r = l.next(); isName(r); r = l.next() {
+	for r = l.next(); isNameStart(r) || isName(r); r = l.next() {
```

Comment:
Condition checks each rune.

Reply:
nope, but i would change `isName` function instead of here
```diff
func isName(v rune) bool {
-	return isAlpha(v) || ...
+	return isName(v) || ...
```